### DISCUSSION TITLE
Walrus | 自由学术论文平台

### DISCUSSION THREAD

**Discussion by AlexWaker** [creatorsand/.github#114] [Ideas]
**一句话 IDEA：**

任何人都可以发表学术论文的平台，所有人都是审稿人

**我的角色：**

全栈。

**其他角色需求（不包含发起人）：**

我也不知道需要啥，有兴趣的可以联系我

**社群昵称：**

微信：十九
GitHub：AlexWaker

**个人简介：**

什么都懂但什么也不懂的学术混子。
休学gap+延毕选手。
放弃算法，准备主攻开发，希望成为全栈独立开发者。

**产品是给谁用的？**

中本聪的《比特币：一种点对点式的电子现金系统》论文发表在互联网上，没有发表在所谓的学术顶会上，但毫无疑问他的论文是被市场、被群众筛选出来的改变世界的论文。
产品给任何拥有想法的人用，类似arXiv。但增加了评论功能，任何人都可以是审稿人，给作者意见以及相互讨论。

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]

**Comment by AlexWaker** [comment]
https://github.com/AlexWaker/Walrus_LHFA
